aptos-node/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "aptos-node"
 description = "Aptos node"
-version = "0.1.0"
+version = "1.4.0"
```

Comment:
Worth calling out that once we do this, we're promising to maintain correctness going forward, for all releases and hotfixes, right? Are we wanting to make that promise -- I worry that this is going to become another thing that we'll forget to update on releases 😢  We don't think `./aptos-node --info` is enough?

Reply:
I leave it to @perryjrandall his thoughts on the releases.

I already do this for the CLI and the SDK for packaged releases every time, and it's fairly standard